lib/jira/commands/tickets.rb

```diff
@@ -1,43 +1,52 @@
 module Jira
   class CLI < Thor
 
-    desc "tickets", "List the tickets of the input username"
+    desc "tickets [--jql]", "List the tickets of the input username (or jql)"
+    method_option :jql, type: :string, default: ''
     def tickets(username=Jira::Core.username)
-      Command::Tickets.new(username).run
+      Command::Tickets.new(username, options[:jql]).run
     end
+
   end
 
   module Command
     class Tickets < Base
 
-      attr_accessor :username
+      attr_accessor :username, :jql
 
-      def initialize(username)
+      def initialize(username, jql)
         self.username = username
+        jql = "assignee=#{username}" if jql.empty?
+        self.jql = jql
       end
 
       def run
         return if username.empty?
         return if metadata.empty?
+        puts "There are no tickets for jql=#{jql}." and return if rows.empty?
+        render_table(header, rows)
+      end
 
-        issues = metadata['issues']
-        if !issues.nil? and issues.count > 0
-          issues.each do |issue|
-            ticket = issue['key']
+      private
```

Comment:
can we have this be outdented? ill configure a linter eventually to catch these automatically